src/components/Checkbox/docs/examples.ts

```diff
@@ -19,3 +19,19 @@ export const exampleWithDisabled = `
 export const exampleWithIndeterminate = `
 <Checkbox onChange={val => console.log(val)} indeterminate label="What should I do 🧐?" />
 `;
+
+export const exampleWithBox = `
+<Checkbox box onChange={val => console.log(val)} label="What's in the box?" />
+`;
+
+export const exampleWithCheckedBox = `
+<Checkbox box checked onChange={val => console.log(val)} label="A checkbox!" />
+`;
+
+export const exampleWithDisabledCheckedBox = `
+<Checkbox box checked disabled onChange={val => console.log(val)} label="I'm a disabled, checked box 😶" />
+`;
+
+export const exampleWithSuffix = `
+<Checkbox box suffix="$1,000" onChange={val => console.log(val)} label="An expansive checkbox." />
+`;
```

Comment:
Great examples 👍 
Let's also add an example, where the container is small, and so the content will wrap as described here: https://zeroheight.com/7sjjzhgo2/p/05a744-05-selections/t/76a987